{{ cookiecutter.project_slug }}/pyproject.toml

```diff
@@ -70,7 +77,7 @@ disallow_untyped_defs = false
 addopts = "-v -p no:warnings --cov={{ cookiecutter.project_slug }} --cov-branch --cov-report=html --cov-report=xml --doctest-modules --ignore={{ cookiecutter.project_slug }}/__main__.py --ignore=docs/"
 
 [tool.ruff]
-target-version = "py312"
+target-version = "py313"
```

Comment:
Good spot. Wish `ruff` was smart enough to figure this out on its own

Reply:
I think ruff needs this because if a package is supposed to support older versions of python but the developer is using the cutting edge, or multiple developers are using different versions, then the files won't actually be compatible with older versions.